tests/bootstrap.php

```diff
@@ -18,4 +18,9 @@
 
 Yii::setAlias('@yiiunit', __DIR__);
 
+if (!class_exists('PHPUnit_Framework_TestCase') && class_exists('PHPUnit\Framework\TestCase')) {
```

Comment:
How about moving it to `compatibility.php`? I'm currently working on tests and implemented custom assertion which needs another BC hack because of `extends \PHPUnit_Framework_Constraint`...